pkg/server/api/logger/v1/service.go

```diff
@@ -0,0 +1,60 @@
+package logger
+
+import (
+	"context"
+
+	loggerv1 "github.com/spiffe/spire-api-sdk/proto/spire/api/server/logger/v1"
+	"github.com/spiffe/spire-api-sdk/proto/spire/api/types"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+type Config struct {
+	DefaultLevel logrus.Level
+}
+
+type Service struct {
+	loggerv1.UnsafeLoggerServer
+
+	DefaultLevel logrus.Level
```

Comment:
If you plumb through the `*logrus.Logger` set up in run.go as suggested, then one possibility is for `New` to record the level of the logger and record that as the "default" instead of passing the default log level through to here.